openml-lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGResources.java

```diff
@@ -43,38 +42,38 @@
     /**
      * SWIG pointer to BoosterHandle.
      */
-    Long swigBoosterHandle;
+    public Long swigBoosterHandle;
```

Comment:
SWIGResources is a helper for internal operations only. Please restore visibility of members and methods.

Reply:
Is there any reason for using `Long` instead of `long`?

Reply:
I think I got the idea, it is because if can be nullable. At least it is shown on `if (this.swigBoosterHandle != null)` on the same class